allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/AllureSupportKaspressoBuilder.kt

```diff
@@ -48,10 +46,81 @@ fun Kaspresso.Builder.addAllureSupport(): Kaspresso.Builder = apply {
             listOf(
                 DumpLogcatTestInterceptor(logcatDumper),
                 ScreenshotTestInterceptor(screenshots),
-                DumpViewsTestInterceptor(viewHierarchyDumper),
-                VideoRecordingTestInterceptor(videos, allureResourcesFilesProvider, videosHolder),
-                MoveReportsInterceptor(instrumentation, dirsProvider, allureResourcesRootDirsProvider, videosHolder, instrumentalDependencyProvider.uiDevice)
+                VideoRecordingTestInterceptor(videos),
+                DumpViewsTestInterceptor(viewHierarchyDumper)
             )
         )
     }
 }
+
+/**
+ * Forces file providers needed fpr foxed allure support
+ */
+fun Kaspresso.Builder.Companion.withForcedAllureSupport(
+    customize: Kaspresso.Builder.() -> Unit = {}
+): Kaspresso.Builder = simple {
+    if (!isAndroidRuntime) {
+        return@simple
+    }
+    customize.invoke(this)
+    val instrumentalDependencyProvider =
+        instrumentalDependencyProviderFactory.getComponentProvider<Kaspresso>(instrumentation)
+    forceAllureSupportFileProviders(instrumentalDependencyProvider)
```

Comment:
Forcing the file providers entities we use for allure support. We override the users entities here to avoid a mess. The name `withForcedAllureSupport` is also descriptive